Public/Src/Cache/ContentStore/Distributed/NuCache/BlobCentralStorage.cs

```diff
@@ -39,6 +40,15 @@ public class BlobCentralStorage : CentralStorage
         /// <inheritdoc />
         protected override Tracer Tracer { get; } = new Tracer(nameof(BlobCentralStorage));
 
+        private static readonly BlobRequestOptions DefaultBlobStorageRequestOptions = new BlobRequestOptions()

```

Comment:
>DefaultBlobStorageRequestOptions [](start = 51, length = 32)

I'm curious, is it a breaking changing option or not? I mean, is it ok to upload the blob with StoreBlobContentMD5 == false and load it with this property as true?

Reply:
I think it is.

---
In reply to: [327371682](https://github.com/microsoft/BuildXL/pull/918#discussion_r327371682) [](ancestors = 327371682)

Reply:
It is an optional feature, as per their documentation.